package/Dockerfile

```diff
@@ -1,19 +1,33 @@
 FROM nginx:1.14.0-alpine
 
 LABEL maintainer "Rancher Labs <support@rancher.com>"
-RUN apk -U --no-cache add bash curl wget ca-certificates tar sysstat\
+ARG ARCH=amd64
+ENV DOCKER_URL_amd64=https://get.docker.com/builds/Linux/x86_64/docker-1.12.3.tgz \
+    DOCKER_URL_arm64=https://github.com/rancher/docker/releases/download/v1.12.3/docker-v1.12.3_arm64.tgz \
+    DOCKER_URL=DOCKER_URL_${ARCH}
+RUN apk -U --no-cache add bash \
+    && rm -f /bin/sh \
+    && ln -s /bin/bash /bin/sh
+RUN apk -U --no-cache add curl wget ca-certificates tar sysstat\
     && mkdir -p /opt/rke-tools/bin /etc/confd \
-    && curl -sLf https://github.com/kelseyhightower/confd/releases/download/v0.16.0/confd-0.16.0-linux-amd64 > /usr/bin/confd \
+    && curl -sLf https://github.com/kelseyhightower/confd/releases/download/v0.16.0/confd-0.16.0-linux-${ARCH} > /usr/bin/confd \
     && chmod +x /usr/bin/confd \
-    && curl -sLf https://get.docker.com/builds/Linux/x86_64/docker-1.12.3.tgz | tar xvzf - -C /opt/rke-tools/bin --strip-components=1 docker/docker \
+    && curl -sLf ${!DOCKER_URL} | tar xvzf - -C /opt/rke-tools/bin --strip-components=1 docker/docker \
     && chmod +x /opt/rke-tools/bin/docker \
     && apk del curl
 
+ENV PORTMAP_URL_amd64=https://github.com/projectcalico/cni-plugin/releases/download/v1.9.1/portmap \
+    PORTMAP_URL_arm64=https://github.com/jianghang8421/plugins/releases/download/v1.9.1-rancher/portmap-arm64 \
```

Comment:
what are we going to do about this?

Reply:
According to the [release](https://github.com/projectcalico/cni-plugin/releases/tag/v1.9.1 ) which the amd64 arch is using. The portmap plugin is built from [this commit](https://github.com/containernetworking/plugins/commit/e8bea554c5ec9a433d4353c03f7f31b068185c1c), not an official release, so I forked the repo `containernetworking/plugins` and built the arm64 portmap from this commit.
I don't know if we can upgrade the portmap to an official release version which support arm64 arch.